relevanceai/dataset_api/dataset.py

```diff
@@ -499,6 +499,28 @@ def all(
             show_progress_bar=show_progress_bar,
         )
 
+    def cat(self, vector_name: Union[str, None] = None, fields: List = []):
+        """
+        Concatenates numerical fields along an axis and reuploads this vector for other operations
+
+        Parameters
+        ----------
+        vector_name: str, default None
+            name of the new concatenated vector field
+        fields: List
+            fields alone which the new vector will concatenate
+        """
+        field_documents = self.get_all_documents(self.dataset_id, select_fields=fields)
```

Comment:
You should use the `apply` function or `pull_update_push`. Loading all your data into memory is often not a good idea.